src/modules/navigator/mission_feasibility_checker.cpp

```diff
@@ -200,33 +214,33 @@ bool MissionFeasibilityChecker::checkHomePositionAltitude(dm_item_t dm_current,
 		}
 
 		/* reject relative alt without home set */
-		if (missionitem.altitude_is_relative && !home_valid && isPositionCommand(missionitem.nav_cmd)) {
+		if (missionitem.altitude_is_relative && !home_valid && MissionBlock::item_contains_position(&missionitem)) {
 
 			warning_issued = true;
 
 			if (throw_error) {
-				mavlink_log_critical(_mavlink_log_pub, "Rejecting mission: No home pos, WP %d uses rel alt", i + 1);
+				mavlink_log_critical(_navigator->get_mavlink_log_pub(), "Mission rejected: No home pos, WP %d uses rel alt", i + 1);
 				return false;
 
 			} else	{
-				mavlink_log_critical(_mavlink_log_pub, "Warning: No home pos, WP %d uses rel alt", i + 1);
+				mavlink_log_critical(_navigator->get_mavlink_log_pub(), "Warning: No home pos, WP %d uses rel alt", i + 1);
 				return true;
 			}
 		}
 
 		/* calculate the global waypoint altitude */
 		float wp_alt = (missionitem.altitude_is_relative) ? missionitem.altitude + home_alt : missionitem.altitude;
 
-		if (home_alt > wp_alt && isPositionCommand(missionitem.nav_cmd)) {
+		if ((home_alt > wp_alt) && MissionBlock::item_contains_position(&missionitem)) {
```

Comment:
Mission feasibility was using a mix of isPositionCommand() and MissionBlock::item_contains_position(). Getting everybody on the same page seemed like a good idea.

For now I think a whitelist is probably better? Should I switch  MissionBlock::item_contains_position?

I was also thinking about coming back to this later and using the mission item's frame instead. What do you think?